data/migrations/V9__disclosure_cand_valid_fec_yr.sql

```diff
@@ -0,0 +1,37 @@
+SET search_path = disclosure, pg_catalog;
```

Comment:
I noticed that this line is in most of the migrations; is `flyway` setting/adding this automatically when generating the migrations?  If so, I wonder why it doesn't just use the fully qualified name in the SQL itself just to be more explicit.  Is there an option that can control or change this?  No worries if not, just curious about it!

Reply:
The migrations were obtained by using `pg_dump` to obtain a schema-only SQL dump from `prod`. `pg_dump` sets `search_path` as opposed to schema-qualifying object names.

Also, this explains how we ended up with ~40 migrations. I started with a ~30,000 line dump file and broke it up into logical groups as well as making it easier to edit, by having smaller files!

Reply:
Ah, okay, thank you for explaining! 🎉